core/payment/src/service.rs

```diff
@@ -491,8 +493,11 @@ mod public {
             return Err(SendError::BadRequest("Invalid payer ID".to_owned()));
         }
 
+        let driver = payment.payment_platform.clone();
```

Comment:
Driver is not the same thing as platform. It doesn't matter for now, but it's a semantic error.

Reply:
fixed via #683